components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -629,24 +648,31 @@ private void sendToFrameworkForLogout(HttpServletRequest request, HttpServletRes
         AuthenticationRequestCacheEntry authenticationRequestCacheEntry =
                 new AuthenticationRequestCacheEntry(authenticationRequest);
         addAuthenticationRequestToRequest(request, authenticationRequestCacheEntry);
+        OIDCSessionManagementUtil.removeOPBrowserStateCookie(request, response);
         sendRequestToFramework(request, response, sessionDataKey, FrameworkConstants.RequestType.CLAIM_TYPE_OIDC);
     }
 
+    private void addOPBSCookieValueToCache(String opBrowserStateCookieValue, OIDCSessionDataCacheEntry cacheEntry) {
```

Comment:
shall we rename this to 'addOPBSCookieValueToCacheEntry'

Reply:
+1. Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1519/commits/dda7bf80cc7f604b87669ae653ed1ac12089cbb2.